doc/ref_model/chapters/chapter02.rst

```diff
@@ -175,7 +175,7 @@ hosted at the Edge. The Telco Edge is a perfect fit for centralized vRAN deploym
 latency requirements.
 
 It is expected that with the technology evolution (e.g. 6G) the use cases will be more demanding. For instance, 
-either to meet less than 1 ms latency, or ultra ultra fast data rate, it will be required to evolve the architecture. 
+either to meet less than 1 ms latency, or ultra fast data rate, it will be required to evolve the architecture. 
```

Comment:
`either to meet less than 1 ms latency, or ultrafast data rate, it will be required to evolve the architecture.`

Reply:
Thank you @tomvanpelt, fixed now.

Reply:
@tomvanpelt, please re-review and approve if you can